enterprise/server/backends/redis_execution_collector/redis_execution_collector.go

```diff
@@ -3,17 +3,22 @@ package redis_execution_collector
 import (
 	"context"
 	"strings"
+	"time"
 
 	"github.com/buildbuddy-io/buildbuddy/enterprise/server/util/redisutil"
 	"github.com/buildbuddy-io/buildbuddy/server/environment"
 	"github.com/go-redis/redis/v8"
 	"google.golang.org/protobuf/proto"
 
 	repb "github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
+	sipb "github.com/buildbuddy-io/buildbuddy/proto/stored_invocation"
 )
 
 const (
-	redisExecutionKeyPrefix = "exec"
+	redisExecutionKeyPrefix  = "exec"
+	redisInvocationKeyPrefix = "inv"
```

Comment:
`inv` is not an obvious abbreviation for invocation so let's write out `invocation`

Reply:
Done.